BragiCode/src/main/java/frc/robot/subsystems/Feeder.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Feeder extends SubsystemBase {
+  /** Creates a new Feeder. */
+  private TalonFX FeederMotor1;
+  private TalonFX FeederMotor2;
+  public FeederEnumState mFeederEnumState;
```

Comment:
mFeederEnumState is not initialized.  What is the starting state for mFeederEnumState?

Reply:
Initialized in the constructor